### DISCUSSION TITLE
Speech To Text Support

### DISCUSSION THREAD

**Discussion by saxena666** [deepgram/community#768] [General help] [answered]
Hi There, i am looking to use Deepgram for Medical scribe use case - for Speech to Text service, Does it give real time transcription on screen as the users speak ? and does it offer multi lingual capability for the real time transcription? 

I also want to deploy the deepgram model to AWS and it has a prerequisite of having an enterprise plan. Thank you

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by davidvonthenen** [reply, marked as answer]
Hi @sabysaxena 

If you enable `interim results`, you can get the transcription as it is happening for you to display on screen:
https://developers.deepgram.com/docs/interim-results

The "pay as you go" offering doesn't require anything to deploy as it's a SaaS based offering. In other words, just make an API call.

If you need self-hosted (as in you run the model locally), you need to speak with someone on the sales team for a contract.
